generated/proto/persist_fs_commitlog.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
If so I need to rename the other to just `fs.proto` to match convention, can do if you are ok with me doing that rename too.